test/unit/uk/gov/hmrc/openidconnect/userinfo/services/UserInfoTransformerSpec.scala

```diff
@@ -193,5 +201,18 @@ class UserInfoTransformerSpec extends UnitSpec with MockitoSugar {
       val userInfoMissingPostCode = userInfo.copy(address = Some(userAddress.copy(formatted = "1 Station Road\nTown Centre\nLondon\nEngland\nUK\nUnited Kingdom\nGB", postal_code = None)), hmrc_enrolments = None, email = None)
       result shouldBe userInfoMissingPostCode
     }
+
+    "not return country code when feature flag is off" in new Setup {
+
+      FeatureSwitch.disable(UserInfoFeatureSwitches.countryCode)
+
+      val scopes = Set("address", "profile", "openid:gov-uk-identifiers", "openid:government_gateway")
+
+      val desUserMissingPostCode = desUserInfo.copy(address = desAddress.copy(postcode = None))
```

Comment:
Is it necessary to force `None` postcode instead of leaving the default one? Test name suggests we're just testing contryCode feature flag.